perfdash/parser.go

```diff
@@ -264,6 +265,29 @@ func parseOperationLatency(latency latencyMetric, operationName string) perftype
 	return perfData
 }
 
+func parseSchedulingHistogram(hist histogramVec, dataItem []perftype.DataItem, operationName string, buildNumber int) {
+	for i := range hist {
+		perfData := perftype.DataItem{Unit: "%", Labels: hist[i].Labels, Data: make(map[string]float64)}
+		perfData.Labels["Operation"] = operationName
+		delete(perfData.Labels, "__name__")
+		count, exists := hist[i].Buckets["+Inf"]
+		if !exists {
+			klog.Errorf("err in build %d: no +Inf bucket: %v", buildNumber, hist)
+			continue
+		}
+		for kBucket, vBucket := range hist[i].Buckets {
+			if kBucket != "+Inf" {
+				if count == 0 {
+					perfData.Data["<= "+kBucket+"s"] = 0
+					continue
+				}
+				perfData.Data["<= "+kBucket+"s"] = float64(vBucket) / float64(count) * 100
```

Comment:
This is calculating percentages in buckets? I don't think this is useful. We are interested in average latency and percentiles.

Reply:
should calculate percentiles from Histograms, rather than change E2eSchedulingLatency from  LatencyMetric to HistogramVec? @liu-cong

Reply:
Yeah end goal is to have latency percentile and average. Changing E2eSchedulingLatency type from LatencyMetric to HistogramVec is implementation detail, which I have an alternative suggestion.

I think my other comment clarifies this. Let me know if you still have questions.

Reply:
thank you, no questions